node/src/Commands.ts

```diff
@@ -1608,6 +1609,42 @@ export function createBitCount(
     return createCommand(RequestType.BitCount, args);
 }
 
+/**
+ * @internal
+ */
+export function createBitPos(
+    key: string,
+    bit: number,
+    start?: number,
+): command_request.Command {
+    const args = [key, bit.toString()];
+
+    if (start) {
+        args.push(start.toString());
+    }
+
+    return createCommand(RequestType.BitPos, args);
+}
+
+/**
+ * @internal
+ */
+export function createBitPosInterval(
```

Comment:
100% you can unite these two, because it is internal API - you fully control arguments.